mongo/cursor_test.go

```diff
@@ -157,5 +157,15 @@ func TestCursor(t *testing.T) {
 			assert.Nil(t, err, "All error: %v", err)
 			assert.True(t, tbc.closed, "expected batch cursor to be closed but was not")
 		})
+
+		t.Run("does not error given interface as parameter", func(t *testing.T) {
+			cursor, err := newCursor(newTestBatchCursor(1, 5), nil)
+			assert.Nil(t, err, "newCursor error: %v", err)
+			var docs interface{}
```

Comment:
Can this line and the one below be condensed into `var docs interface{} = []bson.D{}`?

Reply:
done